ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,222 @@
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc> <persName
+                                    ref="PRS1071dAbbadi"/> describes the handwriting as ʻneatʼ<foreign xml:lang="fr">(ʻfineʼ)</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF"/> glued onto the front board.
+                                </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF"/>  glued onto the inner side of the front board.
+                                    </desc>
+                                </item>
+                            </list>
+                            </additions>
+                        <bindingDesc>
+                            <binding  contemporary="true">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>  
+                                <decoNote xml:id="b2" type="Cover">Leather quarter cover.</decoNote>  
+                                <decoNote xml:id="b3" type="SewingStations" cert="medium">4</decoNote>  
+                                <decoNote xml:id="b4" cert="medium">Two pairs of sewing stations.</decoNote>  
+                                <decoNote xml:id="b5" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                </decoNote>  
+                                <decoNote xml:id="b6" type="SlipCase">According to <persName ref="PRS1071dAbbadi"/>,
+                                    the manuscript has a new traditional two-part leather case.</decoNote>              
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was commissioned by  <persName   ref="PRS1071dAbbadi"/> in Ethiopia. According to him, the text of the manuscript was copied for him ʻto correct the strange omission that had been made of this section in manuscript no. 87.ʼ<foreign xml:lang="fr">(ʻréparer l'étrange oubli qu'on avait fait de cette section dans le manuscrit n. 87ʼ)</foreign>.
```

Comment:
for persons relevant for the manuscript history (scribes, donors, etc etc) it is always good to add a relevant `role` to the `persName `(here `commissioner`)

Space is missing before the translation introduced by the tag <foreign> (a tag does not introduce a space, the current text will be printed as ... `no. 87.ʼ(ʻréparer` ...

Reply:
Ok! thank you!